agent/api/task/task.go

```diff
@@ -709,6 +734,92 @@ func (task *Task) getAllASMSecretRequirements() map[string]apicontainer.Secret {
 	return reqs
 }
 
+// hasLogRouter returns true if the task has a log router container.
+func (task *Task) hasLogRouter() bool {
+	for _, container := range task.Containers {
+		if container.LogRouter != nil { // This is a log router container.
```

Comment:
where is this set?

Reply:
customer will need to specify this in task definition, which will then goes into acs task payload